src/services/codefixes/importFixes.ts

```diff
@@ -178,6 +178,8 @@ namespace ts.codefix {
                 }
             }
 
+            Debug.assert(name !== "default");

```

Comment:
Can you please add a comment explaining why this is true?  Something like "default is a keyword and is not valid as an expression"?